Modules/Mod1A_functional_sim.py

```diff
@@ -2,66 +2,50 @@
 from ontobio.ontol_factory import OntologyFactory
 from ontobio.io.gafparser import GafParser
 from ontobio.assoc_factory import AssociationSetFactory
+from .generic_similarity import GenericSimilarity
 
+URI = 'http://geneontology.org/gene-associations/goa_{group}.gaf.gz'
```

Comment:
The reason for the original conditional below, is that the file naming is not consistent at in the go 'gene_associations' directory http://geneontology.org/gene-associations/

So that pattern won't work for non-human association sets

Reply:
Thanks for catching that @putmantime, by @deepakunni3's suggestion I'm removing the default file altogether.